taprpc/taprootassets.proto

```diff
@@ -668,8 +674,58 @@ message ProofFile {
     string genesis_point = 2;
 }
 
-message ProofVerifyResponse {
+message DecodedProof {
+    // The index depth of the decoded proof, with 0 being the latest proof.
+    int64 proof_at_depth = 1;
+
+    // The total number of proofs contained in the raw proof.
+    int64 number_of_proofs = 2;
+
+    // The asset referenced in the proof.
+    Asset asset = 3;
+
+    // The reveal meta data associated with the proof, if available.
+    AssetMeta meta_reveal = 4;
+
+    // The merkle proof for AnchorTx used to prove its
+    // inclusion within BlockHeader.
+    bytes tx_merkle_proof = 5;
```

Comment:
Is the tx merkle proof really useful without the anchor TX and the block header?
I think we should add those two fields, each just the raw, serialized content.

Reply:
IIUC, that information is already contained within "chain_anchor": {

See https://habibi.cash/asset/AdamCoin5/proof for example

Reply:
Not really, as we only have the block hash in there. But with the block hash someone wanting to verify the proof outside of `tapd` could fetch the whole block and do it that way, so I guess it's okay for now.